test-collateral/generate-policy/src/main.rs

```diff
@@ -576,6 +595,12 @@ fn serialize_identities(arguments: &Arguments) -> Vec<Identity<String>> {
     values
 }
 
+/// Serializes the proxy attestation service certificate (basically reads the
+/// string from the file and returns that
+pub fn serialize_certificate(path: &PathBuf) -> String {
+    return read_to_string(path.to_str().unwrap()).unwrap();
```

Comment:
This can be marked as `inline`.  Also, maybe use `expect` here rather than `unwrap`?